Datadog/E2E/E2EConfig.swift

```diff
@@ -20,27 +20,29 @@ internal class E2EConfig {
     /// Reads Datadog client token authorizing data for 'Mobile - Instrumentation' org.
     static func readClientToken() -> String {
         guard let clientToken = bundle.infoDictionary?[InfoPlistKey.clientToken] as? String, !clientToken.isEmpty else {
-            fatalError(
-                """
-                ✋⛔️ Cannot read `\(InfoPlistKey.clientToken)` from `Info.plist` dictionary.
-                Update `xcconfigs/Datadog.xcconfig` with your own client token obtained on datadoghq.com.
-                You might need to run `Product > Clean Build Folder` before retrying.
-                """
-            )
+            return "client-token"
+//            fatalError(
+//                """
+//                ✋⛔️ Cannot read `\(InfoPlistKey.clientToken)` from `Info.plist` dictionary.
+//                Update `xcconfigs/Datadog.xcconfig` with your own client token obtained on datadoghq.com.
+//                You might need to run `Product > Clean Build Folder` before retrying.
+//                """
+//            )
```

Comment:
This only requires defining following two in `Datadog.local.xcconfig` 😉. Also, real credentials to "Mobile - Integration" org can be used here freely, as the project is configured to use `env:debug` (vs. `env:instrumentation` on CI) so data won't be impacted but and be easily previewed while in development.
```
E2E_RUM_APPLICATION_ID=
E2E_DATADOG_CLIENT_TOKEN=
```